src/index.html

```diff
@@ -6,9 +6,60 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+    
+      <div class="container_logo">
+        <a href="#home">
+          <img src="./images/logo.png" 
+          width="40" 
+          height="40"
+          alt="moyo-logo" 
+          class="nav_logo"
+          >
+        </a>
+      </div>
+    
+      
+      <nav class="header_nav">
+        <ul class="nav__menu">
+          <li>
+            <a href="#" class="is-active nav_link">Apple</a>
+          </li>
+          <li >
```

Comment:
```suggestion
          </li>
          
          <li >
```
fix the code style all over the page. And check it according to the checklist